src/status_im2/common/floating_button_page/view.cljs

```diff
@@ -51,7 +52,8 @@
      :remove-listeners    remove-listeners}))
 
 (defn view
-  [{:keys [header footer]} & children]
+  [{:keys [header footer customization-color gradient-cover?] :or {customization-color :blue}} &
+   children]
```

Comment:
Maybe better not to have a default `blue` customization-color prop here? If someone used this component somewhere the the customization color should be passed, and it wasn't, the component would still render fine. If there is no default is would should a missing prop

Reply:
the gradient cover was crashing without it, but I will move the safety check to that component instead 👍